node/src/accountant/payment_adjuster/mod.rs

```diff
@@ -0,0 +1,2340 @@
+                    service_fee_opt: None
+                },
+                "Current transaction fee balance is not enough to pay a single payment. Number of \
+                canceled payments: 4. Transaction fee per payment: 70,000,000,000,000 wei, while \
+                the wallet contains: 90,000 wei",
+            ),
+            (
+                PaymentAdjusterError::EarlyNotEnoughFeeForSingleTransaction {
+                    number_of_accounts: 5,
+                    transaction_fee_opt: None,
+                    service_fee_opt: Some(ServiceFeeImmoderateInsufficiency{
+                        total_service_fee_required_minor: 6_000_000_000,
+                        cw_service_fee_balance_minor: 333_000_000,
+                    })
+                },
+                "Current service fee balance is not enough to pay a single payment. Number of \
+                canceled payments: 5. Total amount required: 6,000,000,000 wei, while the wallet \
+                contains: 333,000,000 wei",
+            ),
+            (
+                PaymentAdjusterError::EarlyNotEnoughFeeForSingleTransaction {
+                    number_of_accounts: 5,
+                    transaction_fee_opt: Some(TransactionFeeImmoderateInsufficiency{
+                        per_transaction_requirement_minor:  5_000_000_000,
+                        cw_transaction_fee_balance_minor: U256::from(3_000_000_000_u64)
+                    }),
+                    service_fee_opt: Some(ServiceFeeImmoderateInsufficiency{
+                        total_service_fee_required_minor: 7_000_000_000,
+                        cw_service_fee_balance_minor: 100_000_000
+                    })
+                },
+                "Neither transaction fee or service fee balance is enough to pay a single payment. \
+                 Number of payments considered: 5. Transaction fee per payment: 5,000,000,000 wei, \
+                 while in wallet: 3,000,000,000 wei. Total service fee required: 7,000,000,000 wei, \
+                 while in wallet: 100,000,000 wei",
+            ),
+            (
+                PaymentAdjusterError::LateNotEnoughFeeForSingleTransaction {
+                    original_number_of_accounts: 6,
+                    number_of_accounts: 3,
+                    original_service_fee_required_total_minor: 1234567891011,
+                    cw_service_fee_balance_minor: 333333,
+                },
+                "The original set with 6 accounts was adjusted down to 3 due to transaction fee. \
+                The new set was tested on service fee later again and did not pass. Original \
+                required amount of service fee: 1,234,567,891,011 wei, while the wallet contains \
+                333,333 wei."),
+            (
+                PaymentAdjusterError::AllAccountsEliminated,
+                "The adjustment algorithm had to eliminate each payable from the recently urged \
+                payment due to lack of resources.",
```

Comment:
Let's call the error: `RecursionDrainedAllAccounts`.

And the string to: "The payment adjuster wasn't able to compose any combination of payables that can be paid immediately with provided finances."